testproject/ProjectSettings/ProjectSettings.asset

```diff
@@ -672,8 +680,8 @@ PlayerSettings:
   cloudProjectId: 
   framebufferDepthMemorylessMode: 0
   qualitySettingsNames: []
-  projectName: 
-  organizationId: 
+  projectName: relay-stg
+  organizationId: relay-stg
```

Comment:
not needed?

Reply:
Yes, those values are for the cloud services related stuff since we did add relay testing to the project. So for internal testing that is what they need to be, if public devs want to mess around with things this value will automatically get updated with their services info when they login to their services uDash. So for us yes its required.